hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusGetTopicInfoHandler.java

```diff
@@ -145,8 +142,8 @@ private Optional<ConsensusTopicInfo> infoForTopic(
             info.runningHash(Bytes.wrap(meta.runningHash()));
             info.sequenceNumber(meta.sequenceNumber());
             info.expirationTime(meta.expirationTimestamp());
-            if (meta.adminKey() != null) info.adminKey(PbjConverter.toPbj(asKeyUnchecked((JKey) meta.adminKey())));
-            if (meta.submitKey() != null) info.submitKey(PbjConverter.toPbj(asKeyUnchecked((JKey) meta.submitKey())));
+            if (meta.adminKey() != null) info.adminKey(meta.adminKey());
```

Comment:
Does this need the`key == null || key.key().kind().equals(KeyOneOfType.UNSET)` pattern you use [here](https://github.com/hashgraph/hedera-services/blob/24a13c339ae04929be0ba659b59bbf6f445506ec/hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/workflows/PreHandleContext.java#L186)?

Reply:
Thanks @tinker-michaelj  ! Added `!isEmpty` check